### PR TITLE
Tutorial mandi/section verdi cmdline suggested improvements

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

File: docs/pages/2019_IIT_Mandi_India/sections/verdi_cmdline.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ The one marked with an asterisk is the "default" profile, meaning that any ``ver
 .. note::
 
     The output you get may differ.
-    The ``generic`` profile is pre-configured on the virtual machine built for the tutorial (but we are not going to use it here).
+    The ``generic`` profile is pre-configured on the virtual machine and built only for the first part of the tutorial.
```
Comment (CasperWA) [inline docs/pages/2019_IIT_Mandi_India/sections/verdi_cmdline.rst:63]: Is this true? Huh.

File: docs/pages/2019_IIT_Mandi_India/sections/verdi_cmdline.rst [hunk 2]
```diff
@@ -388,20 +388,23 @@ You should be able to rotate the view with the right mouse button.
 
     If you receive some errors, make sure you started your SSH connection with the ``-X`` or ``-Y`` flag.
 
-Alternatively, especially if showing them interactively is too slow over SSH, you can export the content of a structure node in various popular formats such as ``xyz`` or ``xsf``.
-This is achieved by typing in the terminal:
+Alternatively -- especially if viewing the structure over SSH is too slow -- you can export the content of a structure node in various formats, such as ``xyz`` or ``xsf``, and then view them on your local machine.
+For example, to view the file with the ``xcrysden`` application, first export the structure node in the *xsf* format:
```
Comment (CasperWA) [inline docs/pages/2019_IIT_Mandi_India/sections/verdi_cmdline.rst]:
```suggestion
For example, to view the file with the ``xcrysden`` application, first export the structure node in the ``xsf`` format:
```
Just to keep with the formatting in the prior sentence.

File: docs/pages/2019_IIT_Mandi_India/sections/verdi_cmdline.rst [hunk 3]
```diff
@@ -388,20 +388,23 @@ You should be able to rotate the view with the right mouse button.
...
 
 .. code:: bash
 
     # verdi data structure export --format xsf <IDENTIFIER> > <IDENTIFIER>.xsf
     verdi data structure export --format xsf 254e5a86 > 254e5a86.xsf
 
-You can open the generated ``xsf`` file and observe the cell and the coordinates.
-Then, you can then copy ``<IDENTIFIER>.xsf`` from the Amazon machine to your local one and then visualize it, e.g. with ``xcrysden`` (if you have it installed):
+Then **copy** the xsf-file to your local machine and open it with the ``xcrysden`` application:
```
Comment (CasperWA) [inline docs/pages/2019_IIT_Mandi_India/sections/verdi_cmdline.rst]:
```suggestion
Then **copy** the ``xsf``-file to your local machine and open it with the ``xcrysden`` application:
```

File: docs/pages/2019_IIT_Mandi_India/sections/verdi_cmdline.rst (hunk 1 above)
Comment (csadorf) [reply docs/pages/2019_IIT_Mandi_India/sections/verdi_cmdline.rst:63]: You mean that the *generic* profile is no longer used later? It seems so, yes. It actually creates a few problems IMO.

**Review by CasperWA — CHANGES_REQUESTED** [review]
Good suggestions.
Only really some stuff about format consistency. Either way is fine with me, as long as it's consistent.

**Review by CasperWA — APPROVED** [review]
Thanks @csadorf!

**Pull request merged**